index.rst

```diff
@@ -326,33 +326,162 @@ Analysis of unbinned data is clearly needed for pipeline data quality analyses,
+In addition to the Visit database, we recommend recording scalar metrics into a time series database for further monitoring and visualization.
 
-FAFF is assembling a series of use-cases, specifically descriptions of database queries, that will identify the commissioning-specific functionalities required by the project databases.
-This content is currently hosted on `a confluence page <https://confluence.lsstcorp.org/display/LSSTCOM/Use+cases+for+commissioning+databases>`_, but the pertinent content will be merged to this report and/or the use-cases described as part of `Deliverable 1`_.
+Early discussions have yielded data access requirements relevant to the database infrastructure (see :ref:`Data Access <data-access-reqs>`).
 
-Independent of the work describe above, early discussions have already yielded the following requirements on the database infrastructure, with more to come as the work progresses:
 
--  Users require a framework/method that manages the point(s) of access, analogous to the EFD Client (`FAFF-REQ-0055`_).
-   Ideally, users will have the impression all queries are going to a single database, despite what is actually happening on the back-end(s).
-- The database must be available and rapidly synced to at all major data facilities (`FAFF-REQ-0055`_), analogous to what is done for the summit EFD.
-- Summit tooling, including the Scheduler, must have immediate access to the database (`FAFF-REQ-0056`_).
+The Engineering and Facilities Database (EFD)
+"""""""""""""""""""""""""""""""""""""""""""""
+
+The telemetry data from the observatory systems is recorded in the EFD.
+The data is organized by topics following the `T&S XML schema`_ and recorded in InfluxDB, a time series database optimized for queries constrained by time on large datasets.
+
+The EFD is managed by `Sasquatch`_, an RSP service that includes tools like Chronograf for data exploration and visualization and Kapacitor for alerting on the EFD data
+(see also :ref:`Metric databases in Sasquatch <metric-databases>`).
+
+.. _the-butler-registy:
+
+The Butler registry
+"""""""""""""""""""
+
+Rapid Analysis will use the Science Pipelines infrastructure for SFP.
+That infrastructure provides a Butler registry, a database for tracking processing input and output datasets.
+
+The current plan is to have a Butler repository per instrument at the Summit.
+
+Once the exposures are ingested into the Butler "raw" collection, IDs are assigned, and the registry becomes the primary source for obtaining the list of observations.
+
+The Butler registry schema is described in `DMTN-073`_.
+It is designed to support visits composed of multiple exposures.
+If the visit is composed of a single exposure, then the visit table has entries that are essentially duplicates of the science exposures in the exposure table.
+
```

Comment:
I think you can remove this line and make the next line part of the previous paragraph